api/shape/destructure.js

```diff
@@ -333,7 +379,35 @@ export const destructure2 = async (shape, input, ...specs) => {
             } else {
               rest.push(arg);
             }
-          } else if (Shape.isArray(value)) {
+          } else if (Shape.isArray(value) && Shape.isNumber(value[0])) {
+            out.push(value);
+          } else {
+            rest.push(arg);
+          }
+        }
+        output.push(out);
+        break;
+      }
+      case 'segments': {
+        const out = [];
+        for (const arg of args) {
+          let value = await resolve(shape, arg);
+          if (Shape.isSegment(value)) {
+            out.push(value);
+          } else if (Shape.isArray(value) && value.every(Shape.isSegment)) {
+            out.push(...value);
+          } else {
+            rest.push(arg);
+          }
+        }
+        output.push(out);
+        break;
+      }
+      case 'coordinateLists': {
```

Comment:
Has "coordinates" fully replaced "points"?

Reply:
I've started using coordinates for things like [1, 2, 3] as opposed to Point()